src/couch_httpd_misc_handlers.erl

```diff
@@ -185,6 +185,7 @@ handle_config_req(#httpd{method='GET', path_parts=[_, Section, Key]}=Req) ->
     end;
 % POST /_config/_reload - Flushes unpersisted config values from RAM
 handle_config_req(#httpd{method='POST', path_parts=[_, <<"_reload">>]}=Req) ->
+    couch_httpd:validate_ctype(Req, "application/json"),
```

Comment:
Thanks for comments. For consistency reason, I left validate_ctype in _reload request, also reading the body:  _ = couch_httpd:body(Req).